docs/UserGuide.md

```diff
@@ -253,7 +247,8 @@ Note that if an entry has been filtered out / is not currently on the list, then
 Examples:<br>
 `deleteprop 5`: Deletes the fifth property currently visible on the property list.
 
-### Edit a buyer entry in the database: `editbuyer`
+### Edit Commands
+#### Edit a buyer entry in the database: `editbuyer`
 
 Syntax: `editbuyer INDEX {-n NAME} {-ph PHONE} {-e EMAIL} {-a ADDRESS} {-r PRICERANGE} {-c CHARACTERISTICS} {-priority PRIORITY<HIGH, NORMAL, LOW>}`
```

Comment:
```suggestion
Syntax: `editbuyer INDEX {-n NAME} {-ph PHONE} {-e EMAIL} {-a ADDRESS} {-r PRICERANGE} {-c CHARACTERISTICS} {-priority PRIORITY<HIGH, NORMAL, LOW>}`
```
```suggestion
Syntax: `editbuyer INDEX {-n NAME} {-ph PHONE} {-e EMAIL} {-a ADDRESS} {-r PRICERANGE} {-c CHARACTERISTICS} {-pr PRIORITY<HIGH, NORMAL, LOW>}`
```
Same here as above, might need to edit the command summary table at the bottom as well